src/ammer/Library.hx

```diff
@@ -1,4 +1,7 @@
 package ammer;
+#if lua
+typedef Single = Float;
+#end
 
 @:genericBuild(ammer.Library.initLibrary())
 class Library<Const> {
```

Comment:
I added in a typedef for Lua here, but it doesn't feel right.  Should there be a standard set of ANSI numeric types for Lua?  I could add those to the base library just so there's some consistency amongst platforms that commonly extend C native code.

Reply:
That `typedef` definitely should not be here … Was this required to make the build pass? If so, what specifically broke without this? If anything, this kind of mapping might be needed in the stubber.

Reply:
Yeah, your change in the stubber seems right.